src/c99.h

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <limits.h>
+
+typedef size_t _size_t;
+typedef ssize_t _ssize_t;
+typedef intptr_t _intptr_t;
+typedef uintptr_t _uintptr_t;
+
+const int64_t _INT64_MIN = INT64_MIN;
+const int64_t _INT64_MAX = INT64_MAX;
+const uint64_t _UINT64_MAX = UINT64_MAX;
+
+const int_fast64_t _INT_FAST64_MIN = INT_FAST64_MIN;
+const int_fast64_t _INT_FAST64_MAX = INT_FAST64_MAX;
+const uint_fast64_t _UINT_FAST64_MAX = UINT_FAST64_MAX;
+
+const int_least64_t _INT_LEAST64_MIN = INT_LEAST64_MIN;
+const int_least64_t _INT_LEAST64_MAX = INT_LEAST64_MAX;
+const uint_least64_t _UINT_LEAST64_MAX = UINT_LEAST64_MAX;
+
+const intmax_t _INTMAX_MIN = INTMAX_MIN;
+const intmax_t _INTMAX_MAX = INTMAX_MAX;
+const uintmax_t _UINTMAX_MAX = UINTMAX_MAX;
```

Comment:
Pls rename the constants such that they aren't considered reserved names by the C standard ([see here](https://en.cppreference.com/w/c/language/identifier#Reserved_identifiers)). For example, they could be prefixed by `RUST_C99_` or similar...

Reply:
Good point - I renamed them accordingly and removed the unused constants.

Reply:
Thanks!